packages/cycle-scripts/configs/webpack.config.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+// Silence webpack2 deprecation warnings
+// https://github.com/vuejs/vue-loader/issues/666
+process.noDeprecation = true
+
+const { createConfig, defineConstants, env, entryPoint, setOutput, sourceMaps, addPlugins } = require('@webpack-blocks/webpack2');
```

Comment:
Destructuring assignments are not fully supported in node4. I guess we want to keep support to both 4 and 6 for the moment?